db/migrate/20231027041224_add_dfc_name_to_spree_taxons.rb

```diff
@@ -0,0 +1,5 @@
+class AddDfcNameToSpreeTaxons < ActiveRecord::Migration[7.0]
+  def change
+    add_column :spree_taxons, :dfc_name, :string
```

Comment:
I imagined a new table that can attach a DFC link to anything. But it may be good to start simple like this. We can always generalise later and move the data into a new table.